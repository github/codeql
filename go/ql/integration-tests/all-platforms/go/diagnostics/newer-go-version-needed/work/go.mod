go 999.0

module test
