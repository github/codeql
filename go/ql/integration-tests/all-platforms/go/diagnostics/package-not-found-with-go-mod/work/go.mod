go 1.19

require (
        github.com/linode/linode-docs-theme v0.0.0-20220622135843-166f108e1933
)

module test
