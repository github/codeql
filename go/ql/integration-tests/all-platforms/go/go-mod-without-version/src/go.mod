require golang.org/x/net v0.23.0

require golang.org/x/sys v0.18.0 // indirect

module test

