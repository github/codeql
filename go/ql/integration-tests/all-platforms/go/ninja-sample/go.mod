go 1.14

require (
        golang.org/x/net v0.0.0-20200505041828-1ed23360d12c
)

module makesample
