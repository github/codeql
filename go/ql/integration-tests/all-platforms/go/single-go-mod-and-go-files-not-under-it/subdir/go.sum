golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2/go.mod h1:djNgcEr1/C05ACkg1iLfiJU5Ep61QUkGW8qpdssI0+w=
golang.org/x/net v0.0.0-20200505041828-1ed23360d12c h1:zJ0mtu4jCalhKg6Oaukv6iIkb+cOvDrajDH9DH46Q4M=
golang.org/x/net v0.0.0-20200505041828-1ed23360d12c/go.mod h1:qpuaurCH72eLCgpAm/N6yyVIVM9cpaDIP3A8BGJEC5A=
golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a/go.mod h1:STP8DvDyc/dI5b8T5hshtkjS+E42TnysNCUPdjciGhY=
golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd h1:xhmwyvizuTgC2qz7ZlMluP20uW+C3Rm0FD/WLDX8884=
golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
golang.org/x/text v0.3.0/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
