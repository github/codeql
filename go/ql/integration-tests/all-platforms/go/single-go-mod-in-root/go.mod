go 1.14

require golang.org/x/net v0.17.0

module test
