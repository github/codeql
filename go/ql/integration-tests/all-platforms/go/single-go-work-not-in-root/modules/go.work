go 1.19

use (
	./subdir1
	./subdir2
)
