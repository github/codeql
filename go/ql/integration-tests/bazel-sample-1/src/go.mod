go 1.14

require golang.org/x/net v0.33.0

module bazelsample
