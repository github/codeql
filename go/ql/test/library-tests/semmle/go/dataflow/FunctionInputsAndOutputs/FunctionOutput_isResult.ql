import go

from FunctionOutput outp, DataFlow::CallNode c, DataFlow::Node nodeTo
where outp.isResult() and nodeTo = outp.getNode(c)
select c, nodeTo, outp
