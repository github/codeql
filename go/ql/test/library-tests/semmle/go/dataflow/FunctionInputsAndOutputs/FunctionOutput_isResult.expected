| main.go:51:2:51:14 | call to op | main.go:51:2:51:14 | call to op | result |
| main.go:53:2:53:22 | call to op2 | main.go:53:2:53:22 | call to op2 | result |
| main.go:53:14:53:21 | call to bump | main.go:53:14:53:21 | call to bump | result |
| tst2.go:10:9:10:26 | call to NewEncoder | tst2.go:10:9:10:26 | call to NewEncoder | result |
| tst2.go:10:9:10:39 | call to Encode | tst2.go:10:9:10:39 | call to Encode | result |
| tst.go:9:17:9:33 | call to new | tst.go:9:17:9:33 | call to new | result |
